docs/developer-onboarding.md

```diff
@@ -14,3 +20,34 @@ PaaS is built on CloudFoundary and we use the CloudFoundary CLI to interact with
 
 ### Getting familiar with PaaS
 See [PaaS cheatsheet](/docs/paas-cheatsheet.md).
+
+
+### Azure CIP
+
+You will need a CIP account in order to raise a PIM (Privileged Identity Management) request to access certain resources, for example, production environments. Steps on how to register for a CIP account and raise a PIM request
+can be found in the [DfE Technical Guidance](https://technical-guidance.education.gov.uk/infrastructure/hosting/azure-cip/).
+
+## Access to QA, Sandbox and Production
+
+- Navigate to the Support Sign in page: [QA](https://qa.apply-for-teacher-training.service.gov.uk/support/sign-in), [Staging](https://staging.apply-for-teacher-training.service.gov.uk/support/sign-in) and [production](https://www.apply-for-teacher-training.service.gov.uk/support/sign-in)
+- If you already have a DfE Sign In account, login, if you do not click `Create account` and follow the instructions
+- Once you have successfully logged in, you should see an [Apply page](https://qa.apply-for-teacher-training.service.gov.uk/support) stating your `UUID`
```

Comment:
```suggestion
- Once you have successfully logged in, you should see a page stating your DfE Sign-in `UUID`
```

I think It doesn't make sense to link here because that page is different depending on the user's state?